### PR TITLE
Queeries Panel Submission

### PR DESCRIPTION
**Pull request by justdroo**
This is panel submission to Tech Talk DC for, "Queeries", a discussion of professional LGBT issues in tech.

### CODE REVIEW

**Comment by jenpen** [comment]
Approved on my end! We have the full ATX space on Feb 28 which i think will be great for this type of event!

**Comment by PatLeong** [comment]
Hey Drew, great topic, but I have a few questions:

Who else is going to be on the panel (or planning on being on the panel)?  
What is the agenda like (doesn't have to be exact)?
How long is the panel going to go?

I think this is great though, and I can't wait to come.

File: topics.md [hunk 1]
```diff
@@ -16,36 +16,49 @@ This is meant to be a guide -- don't let it overwhelm you. Give us what you've g
...
-* Code of Conduct: I have read, understand, and agree to follow the Tech Talk DC Code of Conduct.
-
-
 ## Accepted Topics
 
 ------------------------------
  
+### Queeries
+  
+  * Session Title: Queeries
```
Comment (PatLeong) [inline topics.md:25]: Suggestion (feel free to ignore): Maybe we can revise the title to include something about Tech?  e.g. "Tech Queeries" "Queeries in/on Tech."

**Comment by justdroo** [comment]
@PatLeong 
**Who else is going to be on the panel (or planning on being on the panel)?**
We're still reaching out to folks, but the goal will be to have 5 panelists

**What is the agenda like (doesn't have to be exact)?**
Some of the general topics we wanted to touch on are:
- What it means to be Queer at work
- Coming out in the workplace
- Tokenization, “poster child”
- Community building
- “Affinity Groups”

**How long is the panel going to go?**
Assuming we had an hour max:
- 5-10 min intro
- 30 min moderated discussion
- 10-20 min audience questions

File: topics.md (hunk 1 above)
Comment (justdroo) [reply topics.md:25]: I'd be open to adding to the name. Let me think about it tonight

**Review by jenpen — APPROVED** [review]

**Review by PatLeong — APPROVED** [review]
I approve too 👍

**Pull request merged**